packages/reactive_image_picker/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: reactive_image_picker
 description: Wrapper around image_picker to work with reactive_forms
-version: 0.8.0
+version: 0.8.1-alpha
```

Comment:
I think we can have just 0.8.1, without `alpha` and we are good to go)

Reply:
Done